epysurv/simulation/seasonal_noise.py

```diff
@@ -1,87 +1,89 @@
 from dataclasses import dataclass
 from typing import Optional, Sequence
 
+import numpy as np
 import pandas as pd
 import rpy2.robjects.packages as rpackages
-from rpy2 import robjects
-
 from epysurv.simulation.base import BaseSimulation
 from epysurv.simulation.utils import add_date_time_index_to_frame, r_list_to_frame
+from rpy2 import robjects
+from scipy.stats import nbinom, poisson
 
 surveillance = rpackages.importr("surveillance")
 
 
 @dataclass
-class SeasonalNoise(BaseSimulation):
-    r"""Generation of a cyclic model of a Poisson distribution as background data for a simulated timevector.
+class SeasonalNoisePoisson(BaseSimulation):
+    r"""Simulation of an endemic time series based on a Poisson distribution.
 
     The mean of the Poisson distribution is modelled as:
-    :math:`\mu = \exp{(amplitude\sin{(frequency \cdot (t + \phi))} + \alpha + \beta \cdot week_t + K \cdot state)}`
-    with :math:`\omega = \pi / 52`
 
-    Attributes
+        :math:`\mu(t) = \exp{(A\sin{(frequency \cdot \omega \cdot (t + \phi))}
+        + \alpha + \beta \cdot t + K \cdot state)}`
+
+    with :math:`\omega = \pi / 52`, :math:`A` being the amplitude, :math:`\beta` the trend parameter, :math:`t`
+    the current week, and :math:`\theta` the seasonal move.
+
+    Parameters
     ----------
     amplitude
-        amplitude (range of sinus), default = 1.
+        amplitude (range of sinus).
     alpha
-        parameter to move along the y-axis (negative values not allowed) with alpha >= amplitude, default = 1.
-    beta
-        regression coefficient, default = 0.
-    phi
-        factor to create seasonal moves (moves the curve along the x-axis), default = 0.
+        parameter to move along the y-axis (negative values not allowed) with `alpha` >= `amplitude`.
     frequency
-        factor to determine the oscillation-frequency, default = 1.
+        factor to determine the oscillation-frequency
+    seasonal_moves
+        seasonal moves (moves the curve along the x-axis).
     seed
-        a seed for the random number generation
-
+        a seed for the random number generation.
+    trend_parameter
+        trend parameter that controls the influence of the current week on :math:`\mu`.
 
     References
     ----------
-    http://surveillance.r-forge.r-project.org/
+        http://surveillance.r-forge.r-project.org/
     """
 
     alpha: float = 1.0
     amplitude: float = 1.0
-    beta: float = 0.0
     frequency: int = 1
-    phi: int = 0
+    seasonal_moves: int = 0
     seed: Optional[int] = None
+    trend_parameter: float = 0.0
```

Comment:
Let's call this just `trend`. Calling it parameter, seems a bit redundant, since all the arguments are in a sense parameters.